src/pallene/pallenelib.lua

```diff
@@ -90,12 +127,209 @@ static lua_Number  pallene_math_log(lua_Integer x, lua_Integer base);
 static lua_Integer pallene_math_modf(lua_State *L, const char* file, int line, lua_Number n, lua_Number* out);
 
 /* Other builtins */
-static TString* pallene_string_char(lua_State *L, const char* file, int line, lua_Integer c);
-static TString* pallene_string_sub(lua_State *L, TString *str, lua_Integer start, lua_Integer end);
+static TString *pallene_string_char(lua_State *L, const char* file, int line, lua_Integer c);
```

Comment:
Were these changes intentional?

Reply:
Probably not. I used some Neovim formatter on the code, maybe those lines also got affected. Shouldn't be an issue, should it?